govc/USAGE.md

```diff
@@ -4291,6 +4358,33 @@ Options:
   -cluster=              Cluster [GOVC_CLUSTER]
 ```
 
+## namespace.ls
+
+```
+Usage: govc namespace.ls [OPTIONS]
+
+Displays the list of namespaces. 
+
+Examples:
+  govc namespace.ls
+
+Options:
+```
+
+## namespace.rm
+
+```
+Usage: govc namespace.rm [OPTIONS]
+
+Deletes a namespace. 
+
+Examples:
+  govc namespace.rm -namespace=test-namespace
+
+Options:
+  -namespace=            The identifier of the namespace.
```

Comment:
Here and other places: why is the namespace an option?
If I don't give a namespace, will it delete all namespaces?